src/Defaults/Trigger/User/UserEmailChanged.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace BracketSpace\Notification\Defaults\Trigger\User;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+
+class UserEmailChanged extends UserTrigger
+{
+	public $changedTo;
+
+	public function __construct()
+	{
+		parent::__construct(
+			'user/email_changed',
+			__(
+				'User email changed',
+				'notification'
+			)
```

Comment:
I can see you took the coding standards from the existing code, but this is a work in progress (as you can see here: https://github.com/BracketSpace/Notification/pull/446 we addressed the line break issues)

The breaks are unnecessary in those functions that can easily fit one line.